src/display.html

```diff
@@ -17,37 +17,54 @@ <h1>Warning</h1>
 				What would you like to do about 
 				<text id="site_url">IF THIS IS RENDERED WE ARE HAVING A BAD DAY</text><text>?</text>
 			</p>
-			<button id="DoNotVisitThisSite">Do Not Visit This Site And Return To Previous Site</button>
-			<br>
-			<button id="VisitAnyways">Visit This Site Anyways</button>
-			<br>
-			<button id="NeverBlockThisSite">Never Block This Site Again</button>
-			<br>
-			<button id="GoToGoogle">Go To Google</button>
-		</div>
+
+			<div class="intercet__actions">
+
+				<button id="DoNotVisitThisSite">Do Not Visit This Site And Return To Previous Site</button>
+				<br>
+				<button id="VisitAnyways">Visit This Site Anyways</button>
+				<br>
+				<button id="NeverBlockThisSite">Never Block This Site Again</button>
+				<br>
+				<button id="GoToGoogle">Go To Google</button>
+			</div>
+			</div>
     </body>
 	<style>
 		body {
 			background-image: url('./images/128_CCC_RED.png');
 			background-attachment: fixed;
 			text-align:center;
+			background-repeat: no-repeat;
```

Comment:
Why lose the tiling?

Reply:
The color of the icons which are not ADA compliant and the style was too similar to the tiling so I just made the site look cleaner overall!